metalearn/metafeatures/metafeatures.py

```diff
@@ -139,65 +122,51 @@ def check_time():
             X, Y, column_types, sample_shape, seed, n_folds
         )
 
-        computed_metafeatures = {name: {self.VALUE_KEY: self.TIMEOUT, self.COMPUTE_TIME_KEY: 0}
-                                 for name in metafeature_ids}
+        computed_metafeatures = {
+            name: self._format_resource(consts.TIMEOUT, 0)
+            for name in metafeature_ids
+        }
         try:
             for metafeature_id in metafeature_ids:
                 self._check_timeout()
                 if verbose:
                     print(metafeature_id)
                 if self._resource_is_target_dependent(metafeature_id) and (
-                    Y is None or column_types[Y.name] == self.NUMERIC
+                    Y is None or column_types[Y.name] == consts.NUMERIC
                 ):
                     if Y is None:
-                        value = self.NO_TARGETS
+                        value = consts.NO_TARGETS
                     else:
-                        value = self.NUMERIC_TARGETS
+                        value = consts.NUMERIC_TARGETS
                     compute_time = None
                 else:
                     value, compute_time = self._get_resource(metafeature_id)
 
-                computed_metafeatures[metafeature_id] = {
-                    self.VALUE_KEY: value,
-                    self.COMPUTE_TIME_KEY: compute_time
-                }
+                computed_metafeatures[metafeature_id] = self._format_resource(value, compute_time)
         except TimeoutError:
             pass
 
         return computed_metafeatures
+    
+    def _format_resource(self, value, compute_time):
+        """Formats the resource data as a dict"""
+        return {
+            consts.VALUE_KEY: value,
+            consts.COMPUTE_TIME_KEY: compute_time
+        }
```

Comment:
This was added as a helper function to replace a hard-coded pattern of dictionary construction that occurred several times throughout this file.